helm/crossplane/README.md

```diff
@@ -167,7 +152,7 @@ replicas: 1
 deploymentStrategy: RollingUpdate
 
 image:
-  repository: crossplane/crossplane
+  repository: xpkg.upbound.io/crossplane/crossplane
```

Comment:
I think we might need to update `retagger` to mirror from `xpkg.upbound.io`. We only do that now for `xpkg.upbound.io/crossplane-contrib/provider-aws`. IIRC they do not push to DockerHub anymore.

Reply:
We have an image for 14 pushed yesterday